internal/handlers/update.go

```diff
@@ -33,16 +34,16 @@ func (h *ServerHandler) handleUploads(w http.ResponseWriter, r *http.Request) {
 	case entities.CounterMetricName:
 		delta, err := strconv.ParseInt(metricValue, 10, 64)
 		if err != nil {
-			h.logger.ErrorContext(r.Context(),
+			sh.logger.ErrorContext(r.Context(),
```

Comment:
See the previous comment: Only errors that result in status code >= 500 should be logged

Please, check that all the parts of your code that log expected errors are fixed